src/internal/views/nunjucks/internal-search/index.njk

```diff
@@ -53,7 +53,7 @@
 {% endmacro %}
 
 {% macro documentLink(doc) %}
```

Comment:
Is it worth renaming the macro to `licenceLink`?

Reply:
Done